### PR TITLE
Updated projects page

### PR DESCRIPTION
**Pull request by Bosch-0**
Closes https://github.com/BitcoinDesign/Guide/issues/649 

- Added more social links for projects (such as links to Telegram / Discord)
- Made descriptions more relevant to what the collaboration is about in collaborations section.
- Updated images
- Put projects in alphabetical order
- Added wallet design improvement project to community projects (pretty basic image used for now, maybe we should design something unique for the project?) 
- Add Join market, Bitcoin Core app, updated Zeus in collaborations
- Lots of other minor changes

[Live preview ](https://deploy-preview-665--sad-borg-390916.netlify.app/projects/)

### CODE REVIEW

**Comment by Bosch-0** [comment]
Should we add Summer of Bitcoin to this PR?

File: projects.md [hunk 1]
```diff
@@ -45,40 +47,46 @@ projects:
         link: https://www.figma.com/community/file/916680391812923706/Bitcoin-Wallet-UI-Kit-(work-in-progress)
       - name: Github
         link: https://github.com/GBKS/bitcoin-wallet-ui-kit
-collaborations:
-  - name: Specter
-    description: A watch-only coordinator for multi-signature and single-key Bitcoin wallets.
+
+  - name: Wallet design improvement project
```
Comment (GBKS) [inline projects.md:51]: @pavlenex just renamed it to `Bitcoin Design Sprints`.

File: projects.md [hunk 2]
```diff
@@ -45,40 +47,46 @@ projects:
...
+    description: Improving Lightning network wallets through applying the design guide, design thinking and problem solving.
     image:
-      url: /assets/images/contribute/specter.png
+      url: /assets/images/contribute/wallt-design-imporvement.png
```
Comment (GBKS) [inline projects.md:54]: Typo in the file name 😱

File: projects.md [hunk 3]
```diff
@@ -45,40 +47,46 @@ projects:
...
       width: 75
       height: 75
     links:
       - name: Site
```
Comment (GBKS) [inline projects.md:58]: `Site` doesn't seem right here as it links to an issue. Maybe `More info`?

File: projects.md [hunk 4]
```diff
@@ -45,40 +47,46 @@ projects:
...
-        link: https://specter.solutions/
-      - name: Collaboration
-        link: https://github.com/BitcoinDesign/Meta/issues/69
-  - name: Zeus
-    description: Manage your Bitcoin and Lightning node on the go.
+        link: https://github.com/BitcoinDesign/Guide/issues/493
```
Comment (GBKS) [inline projects.md:59]:
```suggestion
        link: https://github.com/BitcoinDesign/Meta/issues/244
```

@pavlenex set up a collaboration issue we can link to that will be kept up-to-date.

File: projects.md [hunk 5]
```diff
@@ -11,14 +11,15 @@ projects:
...
+      url: /assets/images/contribute/design-guide.png
       width: 75
       height: 75
     links:
       - name: Site
         link: https://bitcoin.design/guide
       - name: How to contribute
         link: https://bitcoin.design/guide/contribute/
+
   - name: Bitcoin icon set
```
Comment (GBKS) [inline projects.md:23]:
```suggestion
  - name: Bitcoin Icon Set
```

**Review by GBKS — CHANGES_REQUESTED** [review]
Looks good. Just had some minor comments.

**Comment by Bosch-0** [comment]
Updated @GBKS

**Review by pavlenex — APPROVED** [review]
LGTM 🥳

**Review by GBKS — APPROVED** [review]

**Pull request merged**